libecole/tests/src/reward/test-lpiterations.cpp

```diff
@@ -19,18 +19,16 @@ TEST_CASE("LpIterations return the difference in LP iterations between two state
 	auto model = get_solving_model();
 	reward_func.reset(model);
 
-	SECTION("LP iterations are positive") { REQUIRE(reward_func.obtain_reward(model) > 0); }
+	SECTION("LP iterations are positive") { REQUIRE(reward_func.obtain_reward(model) >= 0); }
```

Comment:
It may not be explicit, but in many tests, I assume the resolution is not trivial.
In particular here the Model is on the root note with a strictly positive number of LP iterations.
Testing for a `>=` number of iterations is weaker, and as shown by the mistake you caught in `test_reward.py` leaves it open to mistakes when the number is 0.

Reply:
Ok